meltano.yml

```diff
@@ -23,9 +23,6 @@ plugins:
         kind: string
       - name: "egencia_base_url"
         kind: string
-    config:
```

Comment:
I'm sure we'll cover it in team review, but I'm just curious why this was removed

Reply:
whoops nevermind looks like it got put back in